scripts/python/lib/utilities.py

```diff
@@ -591,7 +592,7 @@ def get_dir(src_dir):
         path (str or None) : Selected path
     """
     rows = 10
-    log = logger.getlogger()
+    logger.getlogger()
```

Comment:
I would just remove this line since the function is not using the logger instance and there isn't a point in instantiating it without a reference.